src/invidious/views/search.ecr

```diff
@@ -2,6 +2,102 @@
 <title><%= search_query.not_nil!.size > 30 ? HTML.escape(query.not_nil![0,30].rstrip(".") + "...") : HTML.escape(query.not_nil!) %> - Invidious</title>
 <% end %>
 
+<h3>
+    <a id="togglefilters" href="javascript:void(0)">[ + ]</a>
+    <%= translate(locale, "filter") %>
+</h3>
+
+<noscript>
+    <style>
+#filters {
+    display: flex !important;
```

Comment:
Please indent this and get rid of the `!important`. I think if the `display: none` is added to `default.css` the styles here overwrite it.